systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,63 @@
+
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Email{% endblock %}
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<h1>E-mail Addresses</h1>
+{% if user.emailaddress_set.all %}
+  <p>The following e-mail addresses are associated with your account:</p>
+  <form  action="{% url 'account_email' %}" method="post">
+    {% csrf_token %}
+    <fieldset class="blockLabels">
+      {% for emailaddress in user.emailaddress_set.all %}
+      <div class="ctrlHolder">
+        <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+          <input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
+          {{ emailaddress.email }}
+          {% if emailaddress.verified %}
+            <span class="verified">Verified</span>
+          {% else %}
+            <span class="unverified">Unverified</span>
+          {% endif %}
+          {% if emailaddress.primary %}<span class="primary">Primary</span>{% endif %}
+        </label>
+      </div>
+      {% endfor %}
+      <div class="buttonHolder">
+        <button class="btn btn-default" type="submit" name="action_primary" >Make Primary</button>
+        <button class="btn btn-default" type="submit" name="action_send" >Re-send Verification</button>
+        <button class="btn btn-default" type="submit" name="action_remove" >Remove</button>
+      </div>
+    </fieldset>
+  </form>
+
+{% else %}
+  <p ><strong>Warning:</strong>You currently do not have any e-mail address set up. You should really add an e-mail address so you can receive notifications, reset your password, etc.</p>
+{% endif %}
+
+
+  
+<form method="post" action="{% url 'account_email' %}" class="add_email">
+  <h2>Add E-mail Address</h2>
+  <hr>
+  {% csrf_token %}
+  {{ form|crispy }}
+  {% if redirect_field_value %}
+    <input type="hidden" name="{{ redirect_field_name }}"
+    value="{{ redirect_field_value }}"/>
+  {% endif %}
+  <br><br>
+
+  <button class="primaryAction btn btn-lg btn-primary btn-block " name="action_add" type="submit">Click to Add E-mail</button>
+</form>
+
+{% endblock %}
+
+
+
```

Comment:
Leave only one empty line by the end of the file. Also remove the extra new lines that appear before the form. One is usually enough to distinguish separate blocks.